### DISCUSSION TITLE
I keep having error on my wordpress dashboard

### DISCUSSION THREAD

**Discussion by dailysflash112** [swoole/swoole-src#4044] [Ideas]
Fatal error: Uncaught Error: Call to undefined function wp() in /home/apexkonn/public_html/wp-blog-header.php:16 Stack trace: #0 /home/apexkonn/public_html/index.php(17): require() #1 {main} thrown in /home/apexkonn/public_html/wp-blog-header.php on line 16

Even after setting it up with the code below

```php
<?php
/**
 * Loads the WordPress environment and template.
 *
 * @package WordPress
 */

if ( ! isset( $wp_did_header ) ) {

	$wp_did_header = true;

	// Load the WordPress library.
	require_once __DIR__ . '/wp-load.php';

	// Set up the WordPress query.
	wp();

	// Load the theme template.
	require_once ABSPATH . WPINC . '/template-loader.php';

}
```
It still won't work.
I need your help guys

**Comment by sy-records** [comment]
I don't understand what you mean, are you using the FastCGI client?

**Reply by dailysflash112** [reply]
Its affecting my website from Cpanel Do you have any Idea on how I can remove it?

**Reply by sy-records** [reply]
What does your question have to do with Swoole?

**Reply by dailysflash112** [reply]
Do you do wordpress?

**Reply by sy-records** [reply]
Of course, I am a WordPress plugin developer. 
But if your question is not related to Swoole, please don't discuss it here.

**Reply by dailysflash112** [reply]
Do you mind If we talk on another social media platform
I'll like to get acquinted with you more.

**Reply by sy-records** [reply]
You can contact me on Twitter.
